common/djangoapps/util/parsing_utils.py

```diff
@@ -0,0 +1,17 @@
+"""
+Utility function for some parsing stuff
+"""
+from xmodule.contentstore.content import StaticContent
+
+
+def course_image_url(course):
+    """
+    Return url of course image.
+    Args:
+        course(CourseDescriptor) : The course id to retrieve course image url.
+    Returns:
+        Absolute url of course image.
+    """
+    loc = StaticContent.compute_location(course.id, course.course_image)
+    url = StaticContent.serialize_asset_key_with_slash(loc)
+    return url
```

Comment:
Since this is only used by the `serializers` module, I don't think this needs to be a separate utility.  You should define this function directly in `serializers.py`